tests/globals.js

```diff
@@ -357,7 +360,12 @@ Object.defineProperty(global, 'modular', {
 });
 
 if (global.Platform.other) {
-  const { initializeApp } = modular;
+  const { initializeApp, setReactNativeAsyncStorage } = modular;
+
+  // Before initializing Firebase set the Async Storage implementation
+  // that will be used to persist user sessions.
+  setReactNativeAsyncStorage(AsyncStorage);
```

Comment:
This wasn't being done for our tests but should have been
Also, there was no modular implementation of it, so I implemented a modular API for it